Dockerfile

```
@@ -33,8 +33,8 @@ RUN \
 FROM golang:1.14 AS release
```

Need to add
```
RUN adduser --system --uid 1234 figment
```
And 
```
RUN chown -R figment /app/
USER 1234
```

---

done @lhopki01 , thanks!

---

@pacmessica  
```suggestion
FROM golang AS release
```
? :) 1.16?